### PR TITLE
Add detailed comments for sample codes

### PR DESCRIPTION
**Pull request by tony-li-avepoint**

### CODE REVIEW

File: samples/powershell/AddSharePointMappings.ps1 [hunk 1]
```diff
@@ -1,16 +1,20 @@
 Try {
-    [System.Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
```
Comment (baron-zhang-avepoint) [inline samples/powershell/AddSharePointMappings.ps1:2]: The fly server is on TLS1.3 now。

File: samples/powershell/AddSharePointMappings.ps1 [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
     [System.Net.HttpWebRequest]::DefaultMaximumErrorResponseLength = -1
     Import-Module ((Split-Path -Parent $MyInvocation.MyCommand.Definition) + "\Common.ps1") -Force
     Import-Module -Name "Fly.Client"
+    #Get the global configuration object to set Fly_API_Endpoint and your access token
     $Configuration = Get-Configuration
     $Configuration["BaseUrl"] = "{Fly_API_Endpoint}"
     $Configuration.AccessToken = "YOUR_BEARER_TOKEN"
+    #Specify the file path of the project mappings to import, only support csv format
     $FilePath = 'C:\Data\50 mapping SP.csv'
```
Comment (baron-zhang-avepoint) [inline samples/powershell/AddSharePointMappings.ps1]: A comment should place to describe the csv format？

File: samples/powershell/GenerateErrorReport.ps1 [hunk 3]
```diff
@@ -1,19 +1,25 @@
 Try {
-    [System.Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
     Import-Module ((Split-Path -Parent $MyInvocation.MyCommand.Definition) + "\Common.ps1") -Force
     Import-Module -Name "Fly.Client"
+    #Get the global configuration object to set Fly_API_Endpoint and your access token
     $Configuration = Get-Configuration
     $Configuration["BaseUrl"] = "{Fly_API_Endpoint}"
     $Configuration.AccessToken = "YOUR_BEARER_TOKEN"
```
Comment (baron-zhang-avepoint) [inline samples/powershell/GenerateErrorReport.ps1:8]: How to get the bearer token？

File: samples/powershell/GenerateErrorReport.ps1 [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+    #Specify the names of the project to generate error report
     $ProjectNames = @('Tony SP', 'zz_test_SP')
+    #Specify the folder path of error report file to download
     $ReportFolderPath = 'C:\Data'
```
Comment (baron-zhang-avepoint) [inline samples/powershell/GenerateErrorReport.ps1]: placehodler maybe better here？  someone may use linux

**Review by baron-zhang-avepoint — CHANGES_REQUESTED** [review]

File: samples/powershell/AddSharePointMappings.ps1 (hunk 1 above)
Comment (tony-li-avepoint) [reply samples/powershell/AddSharePointMappings.ps1:2]: This is for compatibility with lower versions of the system which default protocol type is not Tls 1.2, for Tls 1.3 we will do more research.

File: samples/powershell/AddSharePointMappings.ps1 (hunk 2 above)
Comment (tony-li-avepoint) [reply samples/powershell/AddSharePointMappings.ps1]: Yes, we can remind customers to refer to the csv format when importing mapping in Fly UI

File: samples/powershell/GenerateErrorReport.ps1 (hunk 3 above)
Comment (tony-li-avepoint) [reply samples/powershell/GenerateErrorReport.ps1:8]: First we add comment in Fly user guide about how to get the bearer token, second we provide a single GetIdentityServiceToken.ps1 for customer, we will add comment to refer it

File: samples/powershell/GenerateErrorReport.ps1 (hunk 4 above)
Comment (tony-li-avepoint) [reply samples/powershell/GenerateErrorReport.ps1]: Ok, will use placeholder here.

**Comment by baron-zhang-avepoint** [comment]
we should add the sample code here， why we need another guide to demonstrate this?

thanks

Baron
General Manager, Distinguished DEV Engineer
P: +86.411.8473.6866 | M: 158.4114.3197 | ***@***.******@***.***>
Follow us on Facebook<http://www.facebook.com/pages/Jersey-City-NJ/AvePoint/100618293062?ref=sgm>, Twitter<http://twitter.com/AvePoint_Inc> and LinkedIn<http://www.linkedin.com/profile/view?id=45325986&locale=en_US&trk=tab_pro>!
[cid:be72d728-daff-4384-a27c-339c2d1a6735]
[cid:b50e0967-f8b7-4d8d-bbbb-a17080badb0f]





________________________________

**Comment by tony-li-avepoint** [comment]
Hi @baron-zhang-avepoint, resolved issues in new commit above, please review.

**Pull request merged**